src/BookReader.js

```diff
@@ -473,6 +475,12 @@ BookReader.prototype.getInitialMode = function(params) {
   if (!this.canSwitchToMode(nextMode)) {
     nextMode = this.constMode1up;
   }
+
+  // override defaults mode via `bookreader-defaults` metadata
+  if (null != this.options.defaults) {
+    nextMode = this.options.defaults === 'mode/1up' ? this.constMode1up : this.constMode2up;
```

Comment:
Suggestion, potential blocker: we have possible 3rd option for thumbnail view - can you add support here for it? @nsharma123